### DISCUSSION TITLE
Our real potential from learning to code =>

### DISCUSSION THREAD

**Discussion by D-Upchurch** [nss-day-cohort-47/C47-Discussions#35] [Topix.random]
`<blockquote class="reddit-card" data-card-created="1615303218"><a href="https://www.reddit.com/r/wallstreetbets/comments/m0kf6k/i_programmed_my_leds_to_change_colours_depending/">I programmed my LEDs to change colours depending on GME going up/down</a> from <a href="http://www.reddit.com/r/wallstreetbets">r/wallstreetbets</a></blockquote>
<script async src="//embed.redditmedia.com/widgets/platform.js" charset="UTF-8"></script>`

**Comment by vishus12** [comment]
Just noticed this. I must do this to my RGB's. Diamond hands! 🦍 💎 ✋
